powerhome/playbook
# Add support for height prop in pie charts

#### Screens

![pie-chart-heights](https://user-images.githubusercontent.com/19460/143379789-ee1603fe-827e-4736-a36c-b76ace4a6afc.jpg)

#### Breaking Changes

[No] breaking changes.  Just adding a height prop support for pie charts which can be optional.  Default still set to 400px.

#### Runway Ticket URL

We have changes we need to put in place for Project Launch Dashboard which requires the pie charts to appear in a certain height.  Adjusting the heights however cannot be done since the graphs are rendered on the fly and CSS can't affect the rendered SVG viewBox.

Related Runway Card: [TTNS-723](https://nitro.powerhrg.com/runway/backlog_items/TTNS-723)
Related Huddle: [Project Launch Dashboard](https://huddle.powerapp.cloud/projects/315/handoffs/577)

#### How to test this

_Part 1_
- create a Circle Chart (either in React or in Rails)
- don't specify a height prop.  rendered chart should have a height of 400px in viewBox

_Part 2_
 - create a Circle Chart (either in React or in Rails)
 - pass a specific prop in string (ie., 200 -- for 200px). rendered chart should have a height of 200px in viewBox

#### Checklist:
